setup/MonsterConfig.m

```diff
@@ -207,13 +207,13 @@
             
 			% Properties related to plotting
 			Plot = struct();
-			if obj.SimulationPlot.runtimePlot
-				Plot.Layout = '';
-				Plot.LayoutFigure = '';
-				Plot.LayoutAxes = axes;
-				Plot.PHYFigure = '';
-				Plot.PHYAxes = axes;
-			end
+			%if obj.SimulationPlot.runtimePlot
```

Comment:
Are these not needed anymore?

Reply:
They made an empty figure, where nothing happened.